README.md

```diff
@@ -94,6 +95,8 @@ Object of webpack options. Just `require` in the options from your `webpack.conf
 
 Source maps are always enabled unless explicitly disabled by specifying `devtool: false`.
```

Comment:
I believe this is correct
```suggestion
Source maps are always enabled unless explicitly disabled by specifying `devtool: false` or `mode: "production"`
```

Reply:
probably yes

Reply:
Nope, we are always setting `devtool` even if `mode` is set to `production`. I believe we should **always** set `devtool` unless its explicitly turned off. The code is correct, the `README` is wrong.